config/routes.rb

```diff
@@ -1,14 +1,17 @@
+require 'sidekiq/web'
```

Comment:
Since sidekiq is also used in the jobs and not just here in the routes, would prefer this to be with the other requires in `config/environment.rb`